release.nix

```diff
@@ -36,12 +36,8 @@ let
          packageSet.localPackages;
     # At least the client is broken on darwin for some yarn reason
     plutus-playground = lib.mapAttrs (_: _: linux) 
-        # don't build the docker image on hydra
-        (lib.filterAttrs (n: v: n != "docker") packageSet.plutus-playground);  
     # At least the client is broken on darwin for some yarn reason
     meadow = lib.mapAttrs (_: _: linux) 
-        # don't build the docker image on hydra
-        (lib.filterAttrs (n: v: n != "docker") packageSet.meadow);
```

Comment:
You probably don't want to remove the thing we're filtering over - I think this is a syntax error now!

Reply:
oops